### ISSUE TITLE
Can't assign media keys in custom layers

### ISSUE THREAD

**Issue by p-kolacz** [jlquinn/xbows-driver#13]
I can't assign media keys like `MediaPlay` or `MediaPause` in custom layers. I think it has something to do with 03 suffix needs to be added (https://github.com/jlquinn/xbows-driver/blob/31664d4e65813d6139088e187804de6306ff985d/src/keymap.cc#L34) but I don't know where to go from there.  Because the same problem is for Net keys or mouse buttons I suspect this suffix is not added somewhere. If you can point me in to the right direction I can fix it and send you PR.

**Comment by jlquinn** [comment]
@p-kolacz I had to go back and remind myself about the code and what I had figured out so far.

If you look in doc/spec.md in section `Key code mapping`, you'll see the key id mapping table I worked out.  The 03 suffix refers to the fact that all the media key ids end with 03.

Feel free to ask more questions.  The spec.md captures everything I had figured out about the communication to the keyboard so far.  I never was able to monitor a firmware upgrade to implement that, and there were some other capabilities I hadn't yet fully worked out.
